### ISSUE TITLE
Comments above errors disrupts error line location

### ISSUE THREAD

**Issue by Wipeless** [ImpowerGames/impower#167]
**Describe the bug**
When a comment is above the line of an error, the length of the comment will shift the error line location.

**Steps To Reproduce**
1. Make the following simple script:

```
# RootSection
// A comment above will mess up the error line.
+ Let's try again... > {First|Second|Third}

^
```

**Expected Result**
The sections First, Second, and Third do not exists.  Error lines should appear under these uncreated sections regardless if there's a comment above.

**Actual Result**
The comment and its length influences the error line location.

**Screenshots**
![commentErrors](https://user-images.githubusercontent.com/1523864/161196200-687fabb3-8b1b-4bc4-85b1-4a4bfef251b5.png)

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome
 - Version: 99.0.4844.84

**Additional context**
The example I provided shows choices but this problem can be encountered with any command as long an error exists with a comment above.

**Comment by lovelle-cardoso** [comment]
@Wipeless Fixed by 7e85c71d

**Comment by Wipeless** [comment]
Verified!
